examples/Example.h

```diff
@@ -421,6 +423,24 @@ struct WgWindow : Window
         surfaceDesc.label = "The surface";
         surface = wgpuInstanceCreateSurface(instance, &surfaceDesc);
 
+        // request adapter
+        const WGPURequestAdapterOptions requestAdapterOptions { .nextInChain = nullptr, .compatibleSurface = surface, .powerPreference = WGPUPowerPreference_HighPerformance, .forceFallbackAdapter = false };
+        auto onAdapterRequestEnded = [](WGPURequestAdapterStatus status, WGPUAdapter adapter, char const * message, void * pUserData) { *((WGPUAdapter*)pUserData) = adapter; };
+        wgpuInstanceRequestAdapter(instance, &requestAdapterOptions, onAdapterRequestEnded, &adapter);
+
+        // get adapter and surface properties
+        WGPUFeatureName featureNames[32]{};
+        size_t featuresCount = wgpuAdapterEnumerateFeatures(adapter, featureNames);
+
+        // request device
+        const WGPUDeviceDescriptor deviceDesc { .nextInChain = nullptr, .label = "The device", .requiredFeatureCount = featuresCount, .requiredFeatures = featureNames };
+        auto onDeviceRequestEnded = [](WGPURequestDeviceStatus status, WGPUDevice device, char const * message, void * pUserData) { *((WGPUDevice*)pUserData) = device; };
+        wgpuAdapterRequestDevice(adapter, &deviceDesc, onDeviceRequestEnded, &device);
+
+        // device uncaptured error callback
+        auto onDeviceError = [](WGPUErrorType type, char const* message, void* pUserData) { std::cout << message << std::endl; };
+        wgpuDeviceSetUncapturedErrorCallback(device, onDeviceError, nullptr);
+
```

Comment:
@SergeyLebedkin 
What do you think about we make static helper function to initialize adapter and device?

To remove duplicated code lines and reduce binary size.

I think I'll just copy-paste that codes into WASM binding.

Reply:
Yes, we can do this, if it will be hepfull for you

Reply:
@SergeyLebedkin Please merge `createHandles()`; into `WgWindow()`.

Reply:
@SergeyLebedkin please revert the example code all here and just pass null for device.